ironfish/src/utils/blockchain.ts

```diff
@@ -40,6 +42,23 @@ export function getBlockRange(
   return { start, stop }
 }
 
+export const NATIVE_ASSET_VALUE = {
+  createdTransactionHash: GENESIS_BLOCK_PREVIOUS,
+  identifier: Asset.nativeIdentifier(),
+  metadata: Buffer.from("Native asset of Iron Fish blockchain", 'utf8'),
+  name: Buffer.from("$IRON", 'utf8'),
+  nonce: 0,
+  owner: Buffer.from("Iron Fish", 'utf8'),
+  supply: BigInt(0),
```

Comment:
@rohanjadvani what is the supply number for IRON ?

Reply:
This is defined in our [consensus parameters](https://github.com/iron-fish/ironfish/blob/master/ironfish/src/consensus/consensus.ts#L15), but I would omit this as mentioned above

Reply:
As a note, the consensus parameters can't match this since the supply changes with every block mined.

Reply:
what supply value I should put for Iron?